Kconfig.zephyr-log

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+menu "Logging"
+
+config LOG
+	bool "Enable logging"
```

Comment:
"Enable Logging via Zephyr", this should disable logging via SOF. If this is option = n then the SOF logging is used. Default this will be n until we have all the other pieces in place.

Reply:
Thanks, reworded. and I will keep the default to `y` for test purpose, if this PR get approved, I will force push and change the default value to n.